packages/web/src/store/slices/package.ts

```diff
@@ -0,0 +1,39 @@
+import { createAsyncThunk, createSlice, SerializedError } from '@reduxjs/toolkit';
+import { client, GetPackageInfoOutput } from '../../services/apiClient';
+
+export type PackageInfoSlice = {
+  isLoading: boolean;
+  error?: SerializedError;
+  packageInfo?: GetPackageInfoOutput;
```

Comment:
Could we stick to the pattern in `scans` slice, as in storing each entity under its defined PK in a record? This leads to semi-broken history navigation where data would be refetched upon going back in history.